### PR TITLE
Implement WES logging/info endpoint for local cromwell

### PR DESCRIPTION
**Pull request by mckinsel**
Previously, there wasn't a way to get log information from runs in the local cromwell environment. This PR implements (most of) the `workflows/{workflow_id}` WES endpoint, which returns logs for both the overall workflow and each task.

This also moves the EC2 instances into their own VPC to improve isolation and make it easier to monitor.

### CODE REVIEW

File: external_executors/local_cromwell/local_cromwell.py [hunk 1]
```diff
@@ -218,6 +269,9 @@ def localize_inputs(obj, match_func, localize_func):
     f.write(script)
 END
 
+# Set a maximum run time of 90 minutes to kill stalled jobs.
```
Comment (ambrosejcarr) [inline external_executors/local_cromwell/local_cromwell.py:274]: This probably deserves a comment in whatever documentation you have for this. I imagine it'd be pretty rare to run this on at-scale data, but it'd be nice to tell a user who might pick this up and use it for something weird that it won't work for that.

**Review by ambrosejcarr — APPROVED** [review]
This looks cool. Only one small documentation request, otherwise I think this is good to go. 

I can see why you didn't put this in at the beginning. This is a ton of work to just capture two logs. 😮 

One question: is it possible to see an example of the output?

**Comment by mckinsel** [comment]
Thanks @ambrosejcarr . What did you have in mind for outputs? There is an `outputs` field defined in the WES response to `workflows/{workflow_id}`, but our PR workflows don't generally define any outputs.

**Comment by mckinsel** [comment]
Oh wait, do you mean an example of the response you would get from a request to the endpoint?

**Comment by ambrosejcarr** [comment]
Yep, sorry that I wasn't clear.

File: external_executors/local_cromwell/local_cromwell.py (hunk 1 above)
Comment (mckinsel) [reply external_executors/local_cromwell/local_cromwell.py:274]: Thanks, added to the readme.

**Comment by mckinsel** [comment]
Okay, I've also added an example logging output.

**Pull request merged**